exercises/03_count_byte/main_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "testing"
+
+func TestCountByte(t *testing.T) {
+	for _, tc := range []struct {
+		name   string
+		input  string
+		input2 byte
+		want   int
+	}{
+		{"letter is not repeated", "mango", byte('l'), 0},
+		{"letter is not repeated", "mango", byte('a'), 1},
+		{"letter is repeated twice", "hello", byte('p'), 0},
+		{"letter is repeated twice", "hello", byte('e'), 1},
+		{"letter is repeated twice", "hello", byte('l'), 2},
+		{"Latvian letters", "abčdē", 0xc4, 2},
```

Comment:
Emmm... Nothing is wrong here? U+00C4 is a `'Ä'` letter. Do you see any letter like that in the input string??? You shouldn't be passing bytes above 127, since those require unicode handling!